packages/patternfly-4/react-charts/src/components/ChartBullet/utils/chart-bullet-data.ts

```diff
@@ -197,16 +192,16 @@ export const getQualitativeRangeData = ({
 
   const computedData = datum
     .sort((a: any, b: any) => (invert ? b._y - a._y : a._y - b._y))
-    .map((dataPoint: any, index: number) => {
-      return {
+    .map(
+      (dataPoint: any, index: number) => ({
```

Comment:
Is this required by the linter?

Reply:
Yes the linter flags & changes these based on the [arrow-body-style rule](https://eslint.org/docs/rules/arrow-body-style)